tutorials/applications/Building 1D Rydberg Crystals.ipynb

```diff
@@ -74,7 +74,7 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.    simul = Simulation(seq, sampling_rate=0.6)
</pre><p>I don't know why, but this was giving me an <code>ODEError</code> with the previous sampling_rate, so I had to increase it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Yes, that's the tricky part about that tutorial, since the atoms are so close (in a `MockDevice`) , the terms in the hamiltonian can get really big, so one may have to play with the sampling rate and/or the nsteps